Library/Homebrew/commands.rb

```diff
@@ -31,6 +31,25 @@ module Commands
     "tc"          => "typecheck",
   }.freeze
 
+  INSTALL_FROM_API_FORBIDDEN_COMMANDS = %w[
```

Comment:
Currently, none of them will work because they all require formula files in some way. Some, like `livecheck`and `test` need information that isn't in the formula API so they can't work unless we load a potentially old version from the installed formula file. Some do formula file operations that don't make sense to do if you don't have the taps installed and up to date (e.g. `audit`, `bump-*`, `cat`, `create`, `edit`, `extract`, `formula`, `pr-pull`, `update-python-resources`). And `bottle` requires a formula to be installed from source.

I think most of them could be modified to form when called with formulae from third-party taps. Some might even be able to be modified to work for internal taps too, but the functionality would be different and maybe too confusing. For now, I think getting these to work for third-party taps is a lower priority so we can address those later if we want.